forte/mrdsrg-spin-integrated/mrdsrg.cc

```diff
@@ -126,6 +126,13 @@ void MRDSRG::startup() {
     // test semi-canonical
     semi_canonical_ = check_semi_orbs();
 
+    // Bypass this check if we are doing environment (B) DSRG
```

Comment:
Again, it worries me how much DSRG needs to ask "are we in the embedded case"? All it should need is the Hamiltonian.